apiary.gemspec

```diff
@@ -21,8 +21,8 @@ Gem::Specification.new do |gem|
   gem.test_files    = gem.files.grep(%r{^(test|spec|features)/})
   gem.require_paths = ['lib']
 
-  gem.add_runtime_dependency 'rest-client', '~> 1.8'
-  gem.add_runtime_dependency 'rack', '~> 1.6.4'
+  gem.add_runtime_dependency 'rest-client', '>= 1.8'
```

Comment:
I'd avoid this change, I think it should either be keep as `~> 1.8` or go ahead to `~> 2.0` (or perhaps using a range to support `>= 1.8` and `< 3.0`). Otherwise breaking changed in rest-client might cause apiary-client to break when rest-client 3.0 comes out.